ducktape/tests/reporter.py

```diff
@@ -45,25 +46,25 @@ def format_time(t):
 class SingleResultReporter(object):
     """Helper class for creating a view of results from a single test."""
 
-    def __init__(self, result):
-        self.result = result
+    def __init__(self, datum):
+        self.datum = datum
         self.width = get_terminal_size()[0]
 
     def result_string(self):
         """Stringify single result"""
         result_lines = [
-            "test_id:    %s" % self.result.test_context.test_id,
-            "status:     %s" % pass_fail(self.result.success),
-            "run time:   %s" % format_time(self.result.run_time),
+            "test_id:    %s" % self.datum["test_id"],
+            "status:     %s" % self.datum["status"].upper(),
+            "run time:   %s" % format_time(self.datum["run_time_sec"]),
         ]
 
-        if not self.result.success:
-           # Add summary if the test failed
+        if self.datum["status"].lower() != "pass" and self.datum["error_msg"] is not None:
```

Comment:
Not a big deal, but it looks like we're breaking with previous result data since some of the standard output fields differ? We don't have any way to reload that data so it doesn't matter I guess?